PayPalMessaging/src/main/java/com/braintreepayments/api/PayPalMessagingView.kt

```diff
@@ -83,9 +85,10 @@ class PayPalMessagingView(private val braintreeClient: BraintreeClient) {
                 payPalMessagingListener?.onPayPalMessagingLoading()
             },
             onError = { error ->
-                payPalMessagingListener?.onPayPalMessagingFailure(error)
+                notifyFailure(error)
             },
             onSuccess = {
+                braintreeClient.sendAnalyticsEvent(PayPalMessagingAnalytics.SUCCEEDED.value)
                 payPalMessagingListener?.onPayPalMessagingSuccess()
             }
```

Comment:
I think our other features use a helper method for the success cases too (see [PayPalClient](https://github.com/braintree/braintree_android/blob/25eca78e9e99d3539161022a43b74d7de0d358e2/PayPal/src/main/java/com/braintreepayments/api/PayPalClient.java#L299-L302)). Maybe let's align our pattern across all features?